### DISCUSSION TITLE
Remove Python3 and Keep Python MacOS

### DISCUSSION THREAD

**Discussion by blinkguy182** [pyenv/pyenv#2664] [Q&A] [answered]
I hope I explain this correctly.
MacOS built in version of python was updated to 3.11, this was updated through brew. I also installed python 3.10 using the package available on the python site as I needed it for a project. Now I need 3.9, so I installed through brew along with pyenv. When switching versions it only changes the 'python' version and not the 'python3' version. For some reason only python3 commands work. regular python commands don't. So I removed 3.11, and now the python3 --version command shows 3.10 instead of 3.11. Strange since I didn't install it there to begin with. So now python --version shows 3.9, while python3 --version shows 3.10. when changing the active environment using pyenv global 3.9, the version does show 3.9. When I use python command it still uses 3.10, how is this possible and how do I remove the python3 link from all of the OS and terminal? any help would be great.

**Comment by native-api** [comment, marked as answer]
All installations on PATH outside Pyenv's control are considered the `system` version.

So select `system` and each executable will be whatever is first on PATH after Pyenv's shims.

Of course, if there are multiple such installations, different executables can belong to different installations; as far as Pyenv is concerned, this is still a single "system" version and it doesn't try to second-guess or distinguish them.
